Project 7 - Bankist/index.html

```diff
@@ -0,0 +1,361 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Bankist</title>
+    <link rel="stylesheet" href="css/style.css" />
+    <link
+      rel="icon"
+      type="image/png"
+      sizes="32x32"
+      href="../images/favicon/favicon-32x32.png"
+    />
+  </head>
+
+  <body>
+    <div class="card__title status__message hidden">Login</div>
+
+    <!-- Login -->
+    <section class="login__container">
+      <div class="card sidebar__card">
+        <div class="card__title">Login</div>
+        <form class="form card__body" id="form-login">
+          <div>
+            <input
+              list="users-list"
+              id="user-id"
+              required
+              class="input__group"
+              placeholder="Choose account"
+            />
+            <datalist id="users-list"> </datalist>
+          </div>
+          <div>
+            <input
+              type="number"
+              id="user-pin"
+              class="input__group"
+              placeholder="enter secret pin"
+              required
+            />
+          </div>
+          <div class="btn-container">
+            <button class="btn btn--primary" type="submit" id="btn-login">
+              Sign in
+            </button>
+          </div>
+        </form>
+      </div>
+    </section>
+
+    <!-- Main application -->
+    <div class="wrapper main__app hidden">
+      <nav class="top-navigation">
+        <div class="left-nav">
+          <p class="welcome">Welcome, Sanjay</p>
```

Comment:
@sknepali Class name should be more specific
Let's rename it to `user-greeting`

Reply:
Done